Makefile

```
@@ -24,15 +24,18 @@ GITTAG=$(shell git describe --tags --always)
 GOLDFLAGS?=-extldflags=-zrelro -extldflags=-znow -s -w -X github.com/kubermatic/kubeone/pkg/cmd.version=$(GITTAG) -X github.com/kubermatic/kubeone/pkg/cmd.commit=$(GITCOMMIT) -X github.com/kubermatic/kubeone/pkg/cmd.date=$(BUILD_DATE)
 
 .PHONY: all
-all: install
-
-.PHONY: install
-install: buildenv
-	go install -ldflags='$(GOLDFLAGS)' -v .
+all: build
 
 .PHONY: build
 build: dist/kubeone
 
+dist/kubeone: buildenv
+	go build -ldflags='$(GOLDFLAGS)' -v -o $@ .
+
+.PHONY: install
+install: build
```

```suggestion
install:
```

there is no need to depend on `build` here, as `go install` will also `go build`

---

the reason for that is is described [here](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets). 
```
Compile the program and copy the executables, libraries, and so on to the file names where they should reside for actual use. If there is a simple test to verify that a program is properly installed, this target should run that test.
```

I mean yes, we do not need it.... but if somebody is starting with kubeone it sure will be fascinating that we do not need to compile the program to install it